fastlane/lib/fastlane/actions/get_provisioning_profile.rb

```diff
@@ -55,9 +55,9 @@ def self.author
       # rubocop:disable Lint/MissingKeysOnSharedArea
       def self.output
         [
-          ['SIGH_PROFILE_PATH', 'A path in which export certificates, key and profile'],
-          ['SIGH_PROFILE_PATHS', 'Paths in which export certificates, key and profile'],
-          ['SIGH_UUID', 'UUID (Universally Unique IDentifier)'],
+          ['SIGH_PROFILE_PATH', 'A path in which certificates, key and profile are exported'],
+          ['SIGH_PROFILE_PATHS', 'Paths in which certificates, key and profile are exported'],
+          ['SIGH_UUID', 'UUID (Universally Unique IDentifier) of an application on a device'],
```

Comment:
Does this make sense in the context of sigh/get_provisioning_profile?

Reply:
Maybe the following makes more sense? 👇 
```suggestion
          ['SIGH_UUID', 'UUID (Universally Unique IDentifier) of a provisioning profile'],
```